testsuite/integration/elytron-oidc-client/src/test/resources/org/wildfly/test/integration/elytron/oidc/client/logout/BackChannelLogoutApp-oidc.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Do we have any tests that make use of the new json and subsystem options for specifying a custom logout URI, custom callback URI, etc.?

Reply:
Actually, just realized that I don't see the new subsystem attributes in this PR yet. I know we discussed leaving these out before but now that we're targeting the Preview stability level, they can be added.